cmd/get/account_password/account_password.go

```diff
@@ -0,0 +1,85 @@
+package account_password
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/MakeNowJust/heredoc"
+
+	"github.com/loginradius/lr-cli/api"
+	"github.com/loginradius/lr-cli/cmdutil"
+	"github.com/loginradius/lr-cli/config"
+	"github.com/loginradius/lr-cli/request"
+
+	"github.com/spf13/cobra"
+)
+
+var fileName string
+
+var inpUID string
+
+type Result struct {
+	PasswordHash string `json:"PasswordHash"`
+}
+
+func Newaccount_passwordCmd() *cobra.Command {
+
+	cmd := &cobra.Command{
+		Use:     "account_password",
+		Short:   "get account_password",
+		Long:    `This commmand gets account_password`,
+		Example: heredoc.Doc(`$ lr get account_password --inpUID <inpUID>`),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if inpUID == "" {
+				return &cmdutil.FlagError{Err: errors.New("`email` is required argument")}
+			}
+
+			return get(inpUID)
+
+		},
+	}
+
+	fl := cmd.Flags()
+	fl.StringVarP(&inpUID, "UID", "u", "", "UID")
+
+	return cmd
+}
+
+func get(UID string) error {
+	var resObj *cmdutil.APICred
+	res, err := cmdutil.GetAPICreds()
+	if err == nil {
+		resObj = &cmdutil.APICred{
+			Key:    res.Key,
+			Secret: res.Secret,
+		}
+	} else {
+		resp, err := api.GetSites()
+		if err != nil {
+			return err
+		}
+		resObj = &cmdutil.APICred{
+			Key:    resp.Key,
+			Secret: resp.Secret,
+		}
+		cmdutil.StoreAPICreds(resObj)
+	}
+
+	var UIDstring = UID
```

Comment:
`UID` is already a string no need to again assign it to a variable, we can directly use that.